src/components/InitialData.ts

```diff
@@ -0,0 +1,14 @@
+import { init } from 'i18next';
+
+import { ImportedDataState } from '@excalidraw/excalidraw/types/data/types';
+
+const getInitialData = (elements: any, state: any): ImportedDataState => {
```

Comment:
I imagine that Excalidraw provides a type for a collection of elements as well as for the state. Try to use it.